README.md

```diff
@@ -105,15 +105,31 @@ always in sync, and that one or the other may be slightly more up-to or out-of d
 
 ### `review.tsv`
 Columns:
-- `classCode`: integer: ID of review case class
-- `classShortName`: string (camelCase): describing the review case class
+- `classCode`: integer
+- `classLabel`: string
 - `value`: any: Some form of data to review
 - `comment`: string (optional)
 
-#### 1. `causalD2gButMarkedDigenic`
-This review case involves what would be otherwise considered a valid disease-gene relationship, but for the fact that 
-it quite unusually includes 'digenic' in the label, even though it only had 1 association. OMIM doesn't have a 
+#### 1. D2G Disease-defining but marked digenic
+This review case involves what would be otherwise considered a valid disease-gene (D2G) relationship, but for the fact 
+that it quite unusually includes 'digenic' in the label, even though it only had 1 association. OMIM doesn't have a 
 guaranatee on the data quality of its disease-gene associations marked 'digenic', so for any of these entries, it could 
 be the case that either (a) it is not 'digenic'; OMIM should remove that from the label, and Mondo can make an explicit 
 exception to add the relationship, or could otherwise wait until OMIM fixes the issue and it will automatically be 
 added, or (b) it is in fact 'digenic', and OMIM should add the missing 2nd gene association.
+
+#### 2. D2G: Disease-defining; self-referential
+The unique characteristics of cases of this class are as follows: 
+- Each case has 2 rows in `morbidmap.txt` and are related.
+- Row 1: One row is a typical, valid, disease-defining entry. For the given phenotype MIM in that row, there are no 
+- other rows in `morbidmap.txt` where it appears as a phenotype having an association with another gene.
+  - In all such cases seen thus far as of 2024/11/18, all of these are cancer cases, and the label ends with "somatic".
+  - This entry appears in the Phenotype-Gene Relationships table on the MIM's omim.org/entry page.
+- Row 2: There is a second row where the phenotype in the first row appears as a gene.
+  - For this row, there is no MIM in the phenotype field.
+  - This row does not appear in the Gene-Phenotype Relationships table on the MIM's omim.org/entry page.
+  - This row is self-referential. The label in the Phenotype field is one of the titles of the MIM in the Gene field.
+
+There is a spreadsheet which collates all known cases as of 2024/11/18: [google sheet](
```

Comment:
do you plan to update this spreadsheet or is this just an example spreadsheet of the cases currently found with all future cases being added to `review.tsv`?

Reply:
I don't plan to update it; I left a date there 2024/11/18. If I update the sheet, my plan would be (if I remember) to update that date as well.
There are only 4 cases. Hopefully this will not be common.

Reply:
That's fine. There is no need to update this spreadsheet or even link to it here other than just listing the rows here with the date of the cases that were found.

Reply:
Hmm, I'm a little confused by your last comment. I think this might be a `nit`. I think the example `morbidmap.txt` example just added ([see comment above](https://github.com/monarch-initiative/omim/pull/166#discussion_r1853047244)) together with what's written there is good illustration now.